src/core_loggers/console_logger.ts

```diff
@@ -33,7 +41,10 @@ export class ConsoleLogger extends Logger {
    * @return string
    *     Returns the log message which is used for unit testing purposes.
```

Comment:
To also include something like `and returns void if blah blah blah`?

Reply:
47c55d3